Screens/HomeScreen.js

```diff
@@ -28,12 +28,12 @@ class HomeScreen extends React.Component {
   }
 
   componentWillMount() {
-    FeedStore.addChangeListener(this._updateFeedsFromStore);
+    FeedStore.addChangeListener(this._updateFeedsFromStore.bind(this));
```

Comment:
`this` was not bound to the `HomeScreen`, so it became the store and you set the state on the store ;) Same as below